src/Map/src/Icon/MapIconRendererInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Icon;
+
+interface MapIconRendererInterface
```

Comment:
```suggestion
interface UxIconRendererInterface
```

Reply:
Anyway, is the interface really needed? The method `render(name)` make it really bound to UX Icon, and we do not want the user to replace it with its implementation.